src/App.js

```diff
@@ -1,25 +1,15 @@
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
+import Header from "./components/Header";
+import Section from "./components/Section";
+import Footer from "./components/Footer";
+export default App;
 
 function App() {
```

Comment:
[중요도: 높음] export default를 맨 위로 올린 이유가 뭘까요?

function을 사용하는 컴포넌트의 경우는
```suggestion
export default function App() {
```
위에 처럼 한번에 적어도 무방합니다.

Reply:
https://github.com/codeit-bootcamp-frontend/3-Weekly-Mission/pull/165/commits/03cd01f745111df3e52834073e7da952793cc653

모든 export를 다시 작성했습니다!
굳이 맨 위로 올린 이유는 어디에 export를 적는게 좋을지 아무도 누구도 알려주지 않았기 때문입니다!!

Reply:
아차차